bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/parser/Parser.java

```diff
@@ -17,7 +17,7 @@
  *
  * @author Ana Dimova - Initial Contribution
  */
-public interface Parser {
+public interface Parser<T> {
```

Comment:
hm, doesn't this mean that we need different Parser instances for rules, module types, etc?
I so far assumed that we register Parsers as services that can parse all of these things.